1.19_linux/runtime/cgo/hitsumabushi_syscalls_linux.c

```diff
@@ -265,12 +267,47 @@ int32_t c_write1(uintptr_t fd, void *p, int32_t n) {
     break;
   default:
     fprintf(stderr, "syscall write(%lu, %p, %d) is not implemented\n", fd, p, n);
+    ret = -ENOSYS;
```

Comment:
EBADF?

https://man7.org/linux/man-pages/man2/write.2.html

Reply:
Yes, no strong opinion but it makes sense.